src/views/favorites_panel.dot

```diff
@@ -19,28 +19,43 @@
           <div class="favorite_panel__swipe_window">
             <div class="favorite_panel__swipe_flow--{{= this.favoritePois.length }}">
               {{~ this.favoritePois:poi:i }}
-                <div class="favorite_panel__swipe_element" {{= click(this.go, this, poi) }} id="{{= `favorite_item_${i}` }}">
+                <div class="favorite_panel__item__more_container">
+
+                  <div class="favorite_panel__swipe_element" {{= click(this.go, this, poi) }} id="{{= `favorite_item_${i}` }}">
+
+                    <div class="favorite_panel__item__more_button">
+                      <i class="icon-more-horizontal" {{= click(this.toggleMore, this, i) }}></i>
+                    </div>
+
+
+                    <div class="favorite_panel__item__info">
+                      {{ let ico = IconManager.get(poi); }}
+                      {{? ico }}
+                        <div class="favorite_panel__item__image icon icon-{{= ico.iconClass }}" style="color:{{= ico.color ? ico.color : '#444648' }}"></div>
+                      {{??}}
+                        <div class="favorite_panel__item__image favorite_panel__item__image--small icon-location"></div>
+                      {{?}}
+                      <p class="favorite_panel__item__title">
+                        {{= poi.name ? poi.name : 'default' }}
+                      </p>
+                      <p class="favorite_panel__item__desc">
+                        {{= poi.subClassName ? this.poiSubClass(poi.subClassName) : '' }}
+                      </p>
+                    </div>
 
-                  <div class="favorite_panel__item__actions">
-                    <button class="favorite_panel__remove favorite_panel__action" {{= click(this.del, this, {poi : poi, index : i}) }}>
-                      <i class="icon-x"></i>
-                    </button>
                   </div>
 
-                  <div class="favorite_panel__item__info">
-                    {{ let ico = IconManager.get(poi); }}
-                    {{? ico }}
-                      <div class="favorite_panel__item__image icon icon-{{= ico.iconClass }}" style="color:{{= ico.color ? ico.color : '#444648' }}"></div>
-                    {{??}}
-                      <div class="favorite_panel__item__image favorite_panel__item__image--small icon-location"></div>
-                    {{?}}
-                    <p class="favorite_panel__item__title">
-                      {{= poi.name ? poi.name : 'default' }}
+                   <div class="favorite_panel__item__more" id="favorite_more_{{= i }}">
```

Comment:
Impossible to click on the "delete" button if there is another favorite below: the favorite hover gets the priority.  
Maybe a z-index on this div could help ?